### PR TITLE
APIGOV-27613 - update for the invoke policy to credential type mapping

### PR DESCRIPTION
**Pull request by jcollins-axway**
## Describe the changes

Update documentation around invoke policy mappings

## Deploy preview link

* http://pr-429.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_api_manager/gateway-administation/#invoke-policy-authentication-handling-in-the-discovery-agent
* http://pr-429.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_api_manager/agent-variables/#common-variables-to-both-agents

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by jcollins-axway** [comment]
@vivekschauhan and @sbolosan please take a look

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 7d791ce |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-429.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/connect_api_manager/gateway-administation.md [hunk 1]
```diff
@@ -197,9 +197,16 @@ When discovering and publishing the virtualized APIs with OAuth (External) inbou
...
+When virtualizing a REST API in API Manager, you can configure inbound security to invoke a policy defined in Policy Studio, see [Invoke Policy](https://docs.axway.com/bundle/axway-open-docs/page/docs/apim_administration/apimgr_admin/api_mgmt_virtualize_web/index.html). In cases like these, the Discovery Agent has the ability to map the policy being used for authentication to another supported authentication type.
 
-The Discovery Agent will use the value defined by the `APIMANAGER_INVOKEPOLICY_DEFAULTDESCRIPTION` environment variable if:
+{{< alert title="Note" color="primary" >}}The agent will handle the mapped authentication type as if the proxy were configured to use it. The agent will not validate that the policy uses the mapped authentication type, that responsibility belongs to the policy creator.{{< /alert >}}
+
+To map a policy to an authentication type (OAuth, Basic Auth, API Key) use the following environment variables, incrementing the index as needed.
```
Comment (sbolosan) [inline content/en/docs/connect_manage_environ/connect_api_manager/gateway-administation.md:204]: maybe just one more example where the incremented index is 2

**Review by sbolosan — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connect_api_manager/gateway-administation.md [hunk 2]
```diff
@@ -205,6 +205,7 @@ To map a policy to an authentication type (OAuth, Basic Auth, API Key) use the f
 
 * `APIMANAGER_INVOKEPOLICY_MAPPING_POLICYNAME_1=Invoke API Key` where *Invoke API Key* is the name of the policy attached to the proxy.
 * `APIMANAGER_INVOKEPOLICY_MAPPING_CREDENTIALTYPE_1=APIKey` where *APIKey* is the authentication type that the agent should handle the proxy as. Options are APIKey, Basic, OAuth.
+* Additional mappings would have env vars such as `APIMANAGER_INVOKEPOLICY_MAPPING_POLICYNAME_2` and `APIMANAGER_INVOKEPOLICY_MAPPING_CREDENTIALTYPE_2`, etc.
```
Comment (sbolosan) [inline content/en/docs/connect_manage_environ/connect_api_manager/gateway-administation.md:208]: cool beans

**Review by sbolosan — APPROVED** [review]

**Review by vivekschauhan — APPROVED** [review]

**Review by robertengaxway — APPROVED** [review]
I like the examples provided for Invoke Policy mapping

**Pull request merged**